features/support/api_helpers.rb

```diff
@@ -211,6 +211,12 @@ def create_live_service(framework_slug, lot_slug, supplier_id)
       puts 'Lot slug not recoginsed'
   end
 
+  if lot_slug == 'digital-specialists' and role
+    service_data['services']["#{role}Locations".to_sym] = service_data['services'].delete(:developerLocations)
```

Comment:
It's not 100% clear what's going on here?

I'm guessing the `.delete` returns the value that has been deleted as well as removing it from the `service_data`, so the role supplied as an argument gets the values that used to exist for developer?  And developer role gets removed at the same time?

It might be nice to have a comment to that effect (or if I'm wrong then to explain whatever is going on here).  Why do we need to delete the developer role at all?